projects/packages/jetpack-mu-wpcom/src/features/blog-privacy/blog-privacy.php

```diff
@@ -61,3 +61,35 @@ function robots_txt( string $output, $public ): string {
 }
 
 add_filter( 'robots_txt', __NAMESPACE__ . '\\robots_txt', 12, 2 );
+
+/**
+ * Disable the Open Graph Tags based on the value of either wpcom_public_coming_soon and wpcom_data_sharing_opt_out option.
+ */
+function remove_og_tags() {
+	if ( ! (bool) get_option( 'wpcom_public_coming_soon' ) && ! (bool) get_option( 'wpcom_data_sharing_opt_out' ) ) {
```

Comment:
I don't think `wpcom_data_sharing_opt_out` is the right thing to check here. We don't, for example, turn off search engine access or social media expanded links if that setting is true.

Reply:
The value of `wpcom_data_sharing_opt_out ` is true when the `Prevent third-party sharing for your site` is checked. As a result, I think it would be better to avoid sharing on social media as well.

Reply:
The third parties relevant for this option are Automattic's partners, not social media sites users share their content to. As described in the UI:

> This option will prevent this [public] site’s content from being shared with our licensed network of content and research partners, including those that train AI models.

So turning off OG tags for public sites is outside of the original intent of this setting. I don't think the intent of this setting should expand to include social sharing. If you disagree, could you please make the case in a P2 discussion to get more perspectives involved?

Reply:
Will limit the changes to the Coming Soon 🙂

Reply:
Cool - thanks! Looks like the comment in the lines above needs updating too.

Reply:
Oh thanks! Updated 👍